src/pages/Contributions/LatestContributions/index.tsx

```diff
@@ -31,11 +31,10 @@ interface LatestContributionsProps {
 }
 
 const LatestContributions: FC<LatestContributionsProps> = ({className, contributions}) => {
-  const latestContributions = contributions.slice(0, 20);
-
-  const latestContributionList = useMemo(() => {
-    return orderBy(Object.values(latestContributions), ['created_date'], ['desc']);
-  }, [latestContributions]);
+  const contributionsList = useMemo(() => {
+    return orderBy(Object.values(contributions), ['created_date'], ['desc']);
+  }, [contributions]);
+  const latestContributionsList = contributionsList.slice(0, 50);
```

Comment:
@buckyroberts We are showing `latest` 50 contributions here.
Is this fine or do we show more?

Reply:
why 50 and not all?

- Since these are `Latest Contributions`, not `All Contributions`
- Pagination is not applied here, as we need ALL contributions for the graph so I wasn't really confident rendering ALL contributions elements in the DOM, not sure but it might slow down the page in future